sshtun.go

```diff
@@ -288,9 +336,11 @@ func (tun *SSHTun) listen(localListener net.Listener) error {
 }
 
 func (tun *SSHTun) handle(localConn net.Conn) error {
-	err := tun.addConn()
-	if err != nil {
-		return err
+	if tun.forwardType == Local{
```

Comment:
I think it will be better to remove this condition and add the following:

In `addConn` line 356: `if tun.forwardType == Local && tun.active == 0 {`

In `removeConn` line 375: `if tun.forwardType == Local && tun.active == 0 {`

This way the `active` field will be correctly updated.

Reply:
Yes, that is much better